serverless/discover/about-serverless.adoc

```diff
@@ -26,12 +26,25 @@ You can propagate an event from an xref:../../serverless/discover/knative-event-
 
 The set of supported features, configurations, and integrations for {ServerlessProductName}, current and past versions, are available at the link:https://access.redhat.com/articles/4912821[Supported Configurations page].
 
-// only included for OCP currently until CRD docs are available in OSD
 ifdef::openshift-enterprise[]
+[id="about-serverless-scalability-performance"]
+== Scalability and performance
+
+{ServerlessProductName} has been tested with a configuration of 3 main nodes and 3 worker nodes, each of which has 64 CPUs, 457 GB of memory, and 394 GB of storage each.
```

Comment:
It is the same application that was used for developing this table: https://docs.openshift.com/container-platform/4.3/scalability_and_performance/planning-your-environment-according-to-object-maximums.html

We should also add the footnotes as they are in the above table. Not sure both are needed for our case. 
The Pod count displayed here is the number of test Pods. The actual number of Pods depends on the application’s memory, CPU, and storage requirements.

This was tested on a cluster with 100 worker nodes with 500 Pods per worker node. The default maxPods is still 250. To get to 500 maxPods, the cluster must be created with a maxPods set to 500 using a custom KubeletConfig. If you need 500 user pods, you need a hostPrefix of 22 because there are 10-15 system Pods already running on the node. The maximum number of Pods with attached Persistent Volume Claims (PVC) depends on storage backend from where PVC are allocated. In our tests, only OpenShift Container Storage v4 (OCS v4) was able to satisfy the number of Pods per node discussed in this document.

Reply:
@nainaz 
> It is the same application that was used for developing this table: https://docs.openshift.com/container-platform/4.3/scalability_and_performance/planning-your-environment-according-to-object-maximums.html

This refers to the 4.3 docs; is it the same for newer versions?

> We should also add the footnotes as they are in the above table. Not sure both are needed for our case.

Can you show where precisely you would want to add these or what content is actually relevant by providing a draft doc? I'm not a big fan of adding footnotes to text where they could just be incorporated in the main text. Since we're not using a table format let's maybe think about rewriting the content to just include this info more organically?

Reply:
Good point. I should have been clear about that I meant the information in these footnotes. On second thought, I think we can remove this whole line "Using this configuration, it is possible to allocate 924 pods with a test application before the cluster fails to create new pods." Wdyt @rhuss and @abrennan89 ?

Reply:
OK with me, I'll wait for @rhuss to confirm though

Reply:
I would like to go ahead and finish this. I would like to remove the pod information as it is going to generate more questions than answers. The most important piece of information is 3000 service and average respons time for test application.